test/integration/integration_test.go

```diff
@@ -66,9 +66,9 @@ func TestMain(m *testing.M) {
 	}
 	runner := util.MinishiftRunner{CommandPath: minishiftBinary}
 	if runner.IsCDK() {
-		godogTags += "~minishift-only"
+		godogTags += "~minishift-only ~b2d-only"
 	} else {
-		godogTags += "~cdk-only"
+		godogTags += "~cdk-only ~rhel-only"
```

Comment:
there is also need for logical and: `&&`, however I will make a PR to this branch soon fixing this